src/libraries/System.Security.Cryptography.X509Certificates/tests/CertificateCreation/PrivateKeyAssociationTests.cs

```diff
@@ -426,6 +426,7 @@ public static void AssociatePersistedKey_CNG_DSA()
         }
 
         [Fact]
+        [SkipOnPlatform(TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst, "DSA is not available of iOS/tvOS/MacCatalyst")]
```

Comment:
```suggestion
        [SkipOnPlatform(TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst, "DSA is not available on iOS/tvOS/MacCatalyst")]
```

Reply:
I dropped the s/of/on/ suffix in the message completely. It doesn't really add any value since the platforms are already listed anyway.